### PR TITLE
Review of WebSocket Star

### PR DESCRIPTION
**Pull request by diasdavid**
> **WIP, pushing as I go :)**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,13 +4,13 @@
   "description": "libp2p-webrtc-star without webrtc. Just plain socket.io.",
   "main": "src/index.js",
   "bin": {
-    "websockets-star": "src/rendezvous/bin.js",
+    "websockets-star": "src/rendezvous/bin.js"
```
Comment (mkg20001) [inline package.json]: What about #11 ?

Comment (diasdavid) [reply package.json]: answered in the issue Note, websocket-star doesn't do signaling, it is actually relaying

File: package.json [hunk 2]
```diff
@@ -32,13 +32,18 @@
     "async": "^2.5.0",
     "data-queue": "0.0.2",
     "debug": "^3.0.1",
+    "hapi": "^16.5.2",
```
Comment (mkg20001) [inline package.json]: Can't we have those modules split up in client and server? Putting `hapi` and `socket.io` into this module only makes the client 10x bigger without any benefits for the user.

Comment (diasdavid) [reply package.json]: Ok, we can do that then.

Comment (diasdavid) [reply package.json]: Doing it https://github.com/libp2p/js-libp2p-websocket-star-rendezvous

**Comment by diasdavid** [comment]
I won't have time to look into this again until tomorrow. @mkg20001 if you want to take a stab at continuing modularizing and making sure everything is wired together now that the rendezvous service was factor out, be my guest :)

**Pull request merged**